iOS/Flows/Conversation/Conversation Header/MemberCell.swift

```diff
@@ -40,42 +40,47 @@ class MemberCell: CollectionViewManagerCell, ManageableCell {
 
         self.contentView.clipsToBounds = false
         self.contentView.addSubview(self.personView)
+
+        self.subscribeToUpdates()
     }
 
+    /// A reference to a task for configuring the cell.
+    private var configurationTask: Task<Void, Never>?
+
     func configure(with item: Member) {
-        self.personView.set(person: item.displayable.value)
+        self.configurationTask?.cancel()
```

Comment:
Using a task to fix issues where the member cell is told to configure two different members.